roles/openshift_logging_mux/templates/mux.j2

```diff
@@ -66,6 +66,14 @@ spec:
         - name: muxcerts
           mountPath: /etc/fluent/muxkeys
           readOnly: true
+        - name: filebufferstorage
+{% if openshift_logging_mux_file_buffer_storage_type == 'pvc' %}
```

Comment:
Editing my comment all together, there is no need to specify an if/else or even modify the mountPath. The mountPath can be fixed since this path never changes from the perspective of the container.